.github/workflows/main.yml

```diff
@@ -1,18 +1,27 @@
 name: Build
 
-on: [push]
+on:
```

Comment:
to be able to release using the sbt-softwaremill plugin you'll also need a second publish-release job - see the `main` branch. It also includes release-drafter

Reply:
👍